GitCommands/Git/GitModule.cs

```diff
@@ -757,7 +757,12 @@ public IReadOnlyList<string> GetSortedRefs()
                 "refs/"
             };
 
-            var tree = _gitExecutable.GetOutput(args);
+            string tree = _gitExecutable.GetOutput(args);
+            int warningPos = tree.IndexOf("warning:");
+            if (warningPos >= 0)
```

Comment:
Should the check for "warning" start on the first position i.e. "== 0"?

Reply:
No, that won't work because the output is sorted which makes the warnings appearing last.

Reply:
It should be safe: The colon is no valid character for refs. Isn't it?

Reply:
Correct